web/src/components/TestSpecs/TestSpecs.tsx

```diff
@@ -13,37 +17,67 @@ interface IProps {
 }
 
 const TestSpecs = ({assertionResults, onDelete, onEdit, onOpen, onRevert}: IProps) => {
+  const hiddenElementRef = useRef<HTMLDivElement>(null);
+  const specs = useAppSelector(state => TestSpecsSelectors.selectSpecs(state));
+
+  const getItemSize = useCallback(
+    index => {
+      const item = assertionResults?.resultList?.[index];
```

Comment:
I believe we should add some comments about what is going here, why do we need this etc